birdhouse/scripts/get-components-json.include.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+# Obtain a JSON representation of components enabled on this platform.
+#
+# Expected result should be similar to:
+# {
+#  "components": [
+#    "bird-house/birdhouse-deploy:components/monitoring",
+#    "bird-house/birdhouse-deploy:optional-components/canarie-api-full-monitoring",
+#    "bird-house/birdhouse-deploy:optional-components/all-public-access",
+#    "bird-house/birdhouse-deploy:optional-components/wps-healthchecks",
+#    "bird-house/birdhouse-deploy:optional-components/secure-thredds",
+#    "bird-house/birdhouse-deploy:optional-components/testthredds",
+#    "bird-house/birdhouse-deploy:optional-components/test-weaver",
+#    "bird-house/birdhouse-deploy:components/weaver",
+#    "bird-house/birdhouse-deploy:components/cowbird"
+#  ]
+# }
+#
+
+# default value in case of error or missing definitions
+export BIRDHOUSE_DEPLOY_COMPONENTS_JSON='{"components": []}'
+if [ -z "${EXTRA_CONF_DIRS}" ]; then
+  echo "No components in EXTRA_CONF_DIRS. Components JSON list will be empty!"
+  return
```

Comment:
Why `return`?  You are not in a function !  Just do nothing.

Reply:
Return stops processing of that file at that point, but does not exit the calling script sourcing it. I've tried it with a quick test.

Reply:
Oh I didn't know `return` stopped processing of a sourced file !

I though once a file is sourced, it behaves as if the code is part of the sourcing script so `return` would break out of any functions or loop.

So I tried in `pavics-compose.sh` as well and it works fine.  So let's keep as-is, merge when ready.